src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -1087,6 +1087,85 @@ <h2 class="modal-title">
         </div>
         <!-- End custom validation field -->
 
+        <!-- Short Text -->
```

Comment:
is there an existing block of HTML for short text? can these HTML elements be relocated there? i know this block of code is in need of a big refactor, but we won't be needing it in the React refactor.

Reply:
not as far as I can see... Short Text shares the same elements as Long Text and Number. This would be the first unique element for Short Text.